vmas/simulator/sensors.py

```diff
@@ -123,9 +136,9 @@ def render(self, env_index: int = 0) -> "List[Geom]":
                 xform.set_translation(*self.agent.state.pos[env_index])
                 xform.set_rotation(angle)
                 ray.add_attr(xform)
+                ray.set_color(*self.render_color, alpha=self.alpha)
 
                 ray_circ = rendering.make_circle(0.01)
```

Comment:
```suggestion
                ray_circ = rendering.make_circle(0.01)
                ray_circ.set_color(*self.render_color, alpha=self.alpha)
```
the color of the lidar is the color of its circles, we set it with the same alpha as the lines